applications/conversations/models/class.conversationsmodel.php

```diff
@@ -94,4 +94,44 @@ public function validConversationMember($conversationID, $userID) {
         $conversationMembers = $this->getConversationMembers($conversationID);
         return (in_array($userID, $conversationMembers));
     }
+
+    /**
+     * Notify users when a new message is created.
+     *
+     * @param array|object $conversation
+     * @param array|object $message
+     * @param array $notifyUserIDs
+     */
+    protected function notifyUsers($conversation, $message, $notifyUserIDs) {
+        $conversation = (array)$conversation;
+        $message = (array)$message;
+
+        $activity = [
+            'ActivityType' => 'ConversationMessage',
+            'ActivityUserID' => $conversation['InsertUserID'],
+            'HeadlineFormat' => t('HeadlineFormat.ConversationMessage', '{ActivityUserID,User} sent you a <a href="{Url,html}">message</a>'),
```

Comment:
Since `ActivityUserID` is being set to the user who created the conversation, not the message, notifications are going to say the conversation creator is the one sending everything.

You can test this out by creating a conversation as one user A and sending a message to it as user B. You will likely get a notification on user B that user A sent the message.